app/nostra/account/csv_data_parser.rb

```diff
@@ -0,0 +1,79 @@
+module Nostra
+  module CSVDataParser
```

Comment:
cool module, but any reason not to just use the `CSV` module in the standard library?

Reply:
Yeah, I was having trouble getting it to read properly with the bom-utf8 characters, that by the time I got that sorted out, it just seemed I was already 95% to where I needed to be, but perhaps the library could still be useful.

Reply:
i figured that might be the case after i thought about it more.  i might turn this into a parser specific to this particular type of CSV, unless its more general than that.  you probably know more than me about it

Reply:
I guess with your type of CSV handler in the beginning you're set up for this to parse generally different account CSVs.  if you ever add more formats then things can get broken out as needed.

Reply:
Yeah, I think we won't invest too much time in this as it's not really a main purpose of this application. But we can look at it deeper down the road.